acquia/cli
# CLI-832 | Acli tool export json file with extract characters on the command acli email:configure

**Motivation**

Fixes a bug where customers, when exporting JSON data, saw escaped forward slashes in DNS record values, causing these to not verify correctly when sent to a DNS provider.

**Proposed changes**

Adds the JSON_UNESCAPED_SLASHES constant to the json_encode() method used to dump a JSON representation of DNS record data.

**Testing steps**

1. Follow the [contribution guide](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#building-and-testing) to set up your development environment.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. Run `./bin/acli email:configure` and verify that when exporting a JSON file, forward slashes are no longer escaped.
